### PR TITLE
feat((header.js, hotels.js, speakers.js, sponsors.js, restaurants.js)…

### LINKED ISSUE #108: Add Button Interactions
So I went through the website and realized the buttons don't have any interaction states(hover, pressed, disabled, etc). It would look cooler(because its already cool) if simple animated interactions can be added.
I was wondering if this is something I can work on as my first open-source contribution.

### PR DESCRIPTION
**Pull request by AnselemOdims**
…): add hover styles to buttons

Added hover state styles to buttons across pages/components

"fix #108"

#### What does this PR do
- Adds buttons hover state style across components

#### What are the relevant issues this PR belongs to

#108

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -119,4 +119,4 @@ blockName__elementName_modifierName_modifierValue
 
 ## Financial contributions
 
-We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/osca). Anyone can make a donation or file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
\ No newline at end of file
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/osca). Anyone can make a donation or file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
```
Comment (BolajiAyodeji) [inline CONTRIBUTING.md:122]: Is there a reason why you made a change here?

Comment (BolajiAyodeji) [inline CONTRIBUTING.md:122]: Is there a reason why you made a change here?

**Review by BolajiAyodeji — APPROVED** [review]
Thanks for this PR!

**Comment by BolajiAyodeji** [comment]
@all-contributors please add @AnselemOdims for code

File: CONTRIBUTING.md (hunk 1 above)
Comment (AnselemOdims) [reply CONTRIBUTING.md:122]: I am not sure why that is because I did not make any direct change in that file

**Pull request merged**